process-engine/skeleton
# 🐛 Improve Docker Cleanup after Speedtests

## What did you change?

The speedtests in this repository that are executed by jenkins use docker containers. Because of that, we have a step in our jenkinsfile to clean up after the speedtests are done. This docker cleanup step was not comprehensive up to now.

Every time the speedtests were executed, they left a new dangling docker image, and two new dangling docker volumes.

The changes in this branch make the cleanup step remove dangling images and volumes. This shrinks the diskspace-footprint of running the speedtests from ~**412 MB** to ~**600 KB** per run.

This fixes https://github.com/process-engine/skeleton/issues/42

## How can others test the changes?

- Tell jenkins to run the speedtests
- Check the remaining disk space after every run, and see how it increases by < 1MB, instead of about 400 MB

## PR-Checklist

Please check the boxes in this list after submitting your PR:
